### ISSUE TITLE
Pitch Up/Down Effect commands.

### ISSUE THREAD

**Issue by ilambert** [polyend/TrackerIssues#411]
I'd love to see something similar to what Renoise offers (0Uxx and 0Dxx) in the pattern editor. It's very handy for tape-stop/scratch/rewind type effects.

**Comment by Displeak** [comment]
I would add that a proper Pitch fx instead of the microtune fx would be very useful when working with drum slices

**Comment by barda123** [comment]
I agree! I love the beat slice playback mode but it would be great to be able to pitch individual notes up and down. Not really adding anything here just also want this
